pkgs/tools/security/gnupg/20.nix

```diff
@@ -3,13 +3,13 @@
 
 # Each of the dependencies below are optional.
 # Gnupg can be built without them at the cost of reduced functionality.
-, pinentry ? null, x11Support ? true
+, pinentry ? null, guiSupport ? true
```

Comment:
@matthewbauer I can’t see how this is a problem. I believe, editing ones config to rename one argument (after looking up the new name) is a fairly trivial operation. To make it even easier, we can update the derivation to check for the presence of the old flag and throw an error with the new name.